src/widgets/requirements/ui/requirements.test.tsx

```diff
@@ -1,8 +1,10 @@
 import { screen } from '@testing-library/react';
 import { beforeEach, describe, expect, it } from 'vitest';
-import { Requirements } from './ui/requirements';
+import { Requirements } from './requirements.tsx';
 import { renderWithRouter } from '@/shared/__tests__/utils';
 
+const registerLink = 'https://app.rs.school/registry/mentor';
```

Comment:
Maybe we can use an existing constant LINKS.BECOME_MENTOR from '@/app/const'?

Reply:
Then the test will lose its meaning, since the link text will be taken from the same place.
Am I wrong?

Reply:
Maybe I don't quite understand the purpose of this test. I thought it was just a check for the presence of an attribute href with a correct link. If the link changes globally in the constant LINKS, it should change in the test as well. Am I wrong? ))

Reply:
That's right. But if you put the same constant in the test and in the code, the test will always succeed.
In places where constants are used, using text from the constant will confirm that the change in the constant was not accidental. I stick to this rule.

Reply:
I think I get your point. The purpose of this approach is to avoid accidentally changing the value in a global constant. ok, thanks for the clarification.